src/components/GoXLR.vue

```diff
@@ -1,127 +1,129 @@
-      <div style="display: flex; flex-direction: row; column-gap: 30px">
-        <div>
-          <FileTabs/>
-        </div>
-      </div>
-
-      <div style="height: 25px; background-color: #3b413f"/>
-      <h1 class="sr-only">Device Settings</h1>
-      <Tabs label="Device Settings">
-        <Tab name="Mic">
-          <Mic/>
-        </Tab>
-        <Tab name="Mixer" selected>
-          <ContentContainer>
-            <Mixer/>
-          </ContentContainer>
-        </Tab>
-        <Tab name="Configuration">
-          <ContentContainer>
-            <CenteredContainer>
-              <Faders/>
-              <Cough/>
-            </CenteredContainer>
-          </ContentContainer>
-        </Tab>
-        <Tab v-if="!isDeviceMini()" name="Effects">
-          <EffectsTab/>
-        </Tab>
-        <Tab v-if="!isDeviceMini()" name="Sampler">
-          <ContentContainer>
-            <SamplerTab/>
-          </ContentContainer>
-        </Tab>
-        <Tab name="Lighting">
-          <LightingTab/>
-        </Tab>
-        <Tab name="Routing">
-          <ContentContainer>
-            <Routing/>
-          </ContentContainer>
-        </Tab>
-        <Tab name="System">
-          <ContentContainer>
-            <SystemComponent/>
-          </ContentContainer>
-        </Tab>
-      </Tabs>
-    </template>
-  </div>
+            <div style="height: 25px; background-color: #3b413f" />
+            <h1 class="sr-only">Device Settings</h1>
+            <Tabs label="Device Settings">
+                <Tab name="Mic">
+                    <Mic />
+                </Tab>
+                <Tab name="Mixer" selected>
+                    <ContentContainer>
+                        <Mixer />
+                    </ContentContainer>
+                </Tab>
+                <Tab name="Configuration">
+                    <ContentContainer>
+                        <CenteredContainer>
+                            <Faders />
+                            <Cough />
+                        </CenteredContainer>
+                    </ContentContainer>
+                </Tab>
+                <Tab v-if="!isDeviceMini()" name="Effects">
+                    <EffectsTab />
+                </Tab>
+                <Tab v-if="!isDeviceMini()" name="Sampler">
+                    <ContentContainer>
+                        <SamplerTab />
+                    </ContentContainer>
+                </Tab>
+                <Tab name="Lighting">
+                    <LightingTab />
+                </Tab>
+                <Tab name="Routing">
+                    <ContentContainer>
+                        <Routing />
+                    </ContentContainer>
+                </Tab>
+                <Tab name="System">
+                    <ContentContainer>
+                        <SystemComponent />
+                    </ContentContainer>
+                </Tab>
+            </Tabs>
+            <A11yNotifications />
```

Comment:
Would it be possible to move this down 2 lines so it's available on the device selection page as well (basically push it into the global scope)